openslides_backend/action/relations.py

```diff
@@ -390,16 +389,6 @@ def prepare_result_to_id(
                 )
             else:
                 assert rel_id in remove
-                if (
-                    self.type in ("1:1", "m:1")
-                    and self.reverse_field.on_delete == OnDelete.PROTECT
-                ):
-                    # Hint: There is no on_delete behavior in m:n cases. The reverse
-                    # field is always nullable. We just modifiy the related field list.
-                    raise ActionException(
-                        f"You are not allowed to delete {self.model} {self.id} as "
-                        "long as there are some required related objects."
-                    )
```

Comment:
I removed this since it threw an error when actually the related model was being deleted by the cascade deletion. Structurally, it also makes more sense to have this kind of check in the delete action, since it knows best about which models are deleted and which not. It seems to not have had any unforseen side effects, but please double check if it's ok to remove this here @normanjaeckel.

Reply:
Hm. What about an update case where the field is set to `None` or `[]`. In this case the reverse field is set to `None`. Maybe we want to prevent this case with the OnDelete flag. But maybe this is not correct and we have to use the `required` flag instead.

Reply:
I am not sure I understand you example. If you think we need this (the current tests do not need it, at least), feel free to add it back, but you would've to fix the test again then :D

Reply:
You can leave it out but please open an issue that I have to look at the "required" case again.